widgets/blueprints/src/BlueprintsCatalog.tsx

```diff
@@ -26,7 +27,7 @@ export default function BlueprintsCatalog({
                 <DataSegment.Item
                     selected={item.isSelected}
                     className="fullHeight"
-                    onClick={(event: Event) => {
+                    onClick={(event: React.MouseEvent) => {
```

Comment:
I think
```suggestion
                    onClick={event => {
```
should be enough as the type is already provided by `SegmentItemProps`.

You can remove `import React from 'react';` line then.

Reply:
I've checked that up and you're correct :)
Those changes were implemented by @Blewin, so I didn't try to experiment with them 😁 
Nice catch! 🥳 

As it may a helpful tip for the future, I'm CC'ing you @Blewin, just for the sake of sharing knowledge 🎉